flag_bool_with_inverse.go

```diff
@@ -123,6 +123,14 @@ func (parent *BoolWithInverseFlag) inverseName() string {
 	return parent.InversePrefix + parent.BoolFlag.Name
 }
 
+func (parent *BoolWithInverseFlag) inversePrefix() string {
```

Comment:
Can you add some test cases for this ? i.e setting InversePrefix to something else ?